packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -39,46 +39,19 @@ export type ColumnKey =
   | 'stocktakeNumber'
   | 'description'
   | 'stocktakeDatetime'
-  | 'monthlyConsumption'
-  | 'previousStockOnHand'
-  | 'calculatedQuantity'
-  | 'requestedQuantity'
-  | 'forecastMethod';
+  | 'monthlyConsumption';
 
 const getColumnLookup = <T extends DomainObject>(): Record<
   ColumnKey,
   ColumnDefinition<T>
 > => ({
-  requestedQuantity: {
-    key: 'requestedQuantity',
-    label: 'label.requested-quantity',
-    align: ColumnAlign.Right,
-    width: 100,
-  },
   monthlyConsumption: {
     key: 'monthlyConsumption',
     label: 'label.average-monthly-consumption',
     align: ColumnAlign.Left,
     width: 100,
   },
-  previousStockOnHand: {
-    key: 'previousStockOnHand',
-    label: 'label.stock-on-hand',
-    align: ColumnAlign.Left,
-    width: 125,
-  },
-  calculatedQuantity: {
-    key: 'calculatedQuantity',
-    label: 'label.forecast-quantity',
-    align: ColumnAlign.Left,
-    width: 125,
-  },
-  forecastMethod: {
-    key: 'forecastMethod',
-    label: 'label.forecast-method',
-    align: ColumnAlign.Left,
-    width: 55,
-  },
+
```

Comment:
I removed these columns as they're only really used in request reqs, so doesn't seem useful to put them in a shared space